units/patch_intent.go

```diff
@@ -277,6 +278,24 @@ func (j *patchIntentProcessor) finishPatch(ctx context.Context, patchDoc *patch.
 			}
 		}
 	}
+	alias := j.intent.GetAlias()
+
+	if alias != "" {
+		validAlias := false
+		aliases, err := model.FindAliasInProjectOrPatchedConfig(pref.Id, alias, patchDoc.PatchedProjectConfig)
```

Comment:
Can you verify that this error is visible in the CLI if the alias is invalid, and can we move this logic to a helper function?

Reply:
<img width="902" alt="Screen Shot 2022-01-20 at 6 01 00 PM" src="https://user-images.githubusercontent.com/19805673/150435782-c46f6d76-f464-4b02-a3f3-6dfa00b831fe.png">
<img width="905" alt="Screen Shot 2022-01-20 at 6 01 10 PM" src="https://user-images.githubusercontent.com/19805673/150435794-cf5c64fc-28b8-43cf-a36b-d290b224a7e4.png">

'test2' is defined in the yaml and thus works, 'test' is invalid and we see such in the CLI. I'll change the error to say that the alias isn't specified on the project because it actually doesn't mean it's "invalid" if we get this error.